docs/openapi.yaml

```diff
@@ -48,7 +48,19 @@ tags:
   - name: Index Querying
     description: |
       Get data from the AR.IO Gateway index using GQL
+  - name: Admin
+    description: |
+      Access several password protected features and functions specific to your AR.IO Gateway.  
+
+
+      Input your ADMIN_API_KEY using the green "Authorize" button at the top of this page, or by clicking on the "lock" icon next to any password protected endpoint.
```

Comment:
Good tip, but probably something that's more appropriate in an admin guide. The API docs should remain client agnostic.